_posts/2021-04-07-upgrade-improvements.md

```diff
@@ -5,7 +5,7 @@ date: 2021-04-07
 author: jakub_stejskal
 ---
 The upgrade of Strimzi and Kafka is not a trivial process.
-In Strimzi 0.22 we introduced several improvements to make the process easier.
+In Strimzi 0.22 and 0.24 we introduced several improvements to make the process easier.
```

Comment:
I think this makes it confusing. I would probably list just the last version.

Reply:
Ye, so let's go with 0.24?

Reply:
Yeah, I guess.